pkg/scanner/local/scan_test.go

```diff
@@ -147,411 +147,412 @@ func TestScanner_Scan(t *testing.T) {
+		//		},
+		//		Returns: ApplierApplyLayersReturns{
+		//			Detail: ftypes.ImageDetail{
+		//				OS: &ftypes.OS{
+		//					Family: "alpine",
+		//					Name:   "3.11",
+		//				},
+		//				Packages: []ftypes.Package{
+		//					{
+		//						Name:    "musl",
+		//						Version: "1.2.3",
+		//						LayerID: "sha256:ebf12965380b39889c99a9c02e82ba465f887b45975b6e389d42e9e6a3857888",
+		//					},
+		//				},
+		//			},
+		//		},
+		//	},
+		//	ospkgDetectExpectations: []OspkgDetectorDetectExpectation{
+		//		{
+		//			Args: OspkgDetectorDetectArgs{
+		//				OsFamily: "alpine",
+		//				OsName:   "3.11",
+		//				Pkgs: []ftypes.Package{
+		//					{
+		//						Name:    "musl",
+		//						Version: "1.2.3",
+		//						LayerID: "sha256:ebf12965380b39889c99a9c02e82ba465f887b45975b6e389d42e9e6a3857888",
+		//					},
+		//				},
+		//			},
+		//			Returns: OspkgDetectorDetectReturns{
+		//				Err: errors.New("error"),
+		//			},
+		//		},
+		//	},
+		//	wantErr: "failed to scan OS packages",
+		//},
+		//{
+		//	name: "sad path: libDetector.Detect returns an error",
+		//	args: args{
+		//		target:   "alpine:latest",
+		//		layerIDs: []string{"sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10"},
+		//		options:  types.ScanOptions{VulnType: []string{"library"}},
+		//	},
+		//	applyLayersExpectation: ApplierApplyLayersExpectation{
+		//		Args: ApplierApplyLayersArgs{
+		//			LayerIDs: []string{"sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10"},
+		//		},
+		//		Returns: ApplierApplyLayersReturns{
+		//			Detail: ftypes.ImageDetail{
+		//				OS: &ftypes.OS{
+		//					Family: "alpine",
+		//					Name:   "3.11",
+		//				},
+		//				Packages: []ftypes.Package{
+		//					{
+		//						Name:    "musl",
+		//						Version: "1.2.3",
+		//						LayerID: "sha256:ebf12965380b39889c99a9c02e82ba465f887b45975b6e389d42e9e6a3857888",
+		//					},
+		//				},
+		//				Applications: []ftypes.Application{
+		//					{
+		//						Type:     "bundler",
+		//						FilePath: "/app/Gemfile.lock",
+		//						Libraries: []ftypes.LibraryInfo{
+		//							{
+		//								Library: dtypes.Library{Name: "rails", Version: "6.0"},
+		//								LayerID: "sha256:9bdb2c849099a99c8ab35f6fd7469c623635e8f4479a0a5a3df61e22bae509f6",
+		//							},
+		//						},
+		//					},
+		//				},
+		//			},
+		//		},
+		//	},
+		//	libDetectExpectations: []LibraryDetectorDetectExpectation{
+		//		{
+		//			Args: LibraryDetectorDetectArgs{
+		//				FilePath: "/app/Gemfile.lock",
+		//				Pkgs: []ftypes.LibraryInfo{
+		//					{
+		//						Library: dtypes.Library{Name: "rails", Version: "6.0"},
+		//						LayerID: "sha256:9bdb2c849099a99c8ab35f6fd7469c623635e8f4479a0a5a3df61e22bae509f6",
+		//					},
+		//				},
+		//			},
+		//			Returns: LibraryDetectorDetectReturns{
+		//				Err: errors.New("error"),
+		//			},
+		//		},
+		//	},
+		//	wantErr: "failed to scan application libraries",
+		//},
```

Comment:
Didn't you fix them, yet?

Reply:
Si, I just forget to take the commented section out. I've fixed it.